devices/SparkfunLCD/SparkfunLCD.cs

```diff
@@ -0,0 +1,763 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.SparkfunLCD
+{
+    using System;
+    using System.Device.I2c;
+    using System.Threading;
+    using nanoFramework.Hardware.Esp32;
+
+    /// <summary>
+    /// LCD library for SparkFun RGB Serial Open LCD display (sizes 20x4 or 16x2) with I2C connection
+    /// for product information see https://www.sparkfun.com/products/16398
+    /// code based on https://github.com/sparkfun/OpenLCD
+    /// </summary>
+    public class SparkfunLCD : IDisposable
```

Comment:
Respectfully the device is a Liquid Crystal Display abbreviated LCD, the name SpakfunLCD is already CamelCase. The English language does not recognize Lcd, no such word.

Reply:
Yes but for abbreviation, we do consider them as words. So in DoSomethingHttpRelated, HTTP is written Http.
Same will go for LCD that you'll write Lcd in that case.
See the example of CharacterLcd. That's a convention. We're using it here and in .NET IoT https://github.com/dotnet/iot/.
So please adjust to our convention. If you are using Visual Studio, use the rename option of the file, it will propose to rename the classes and take care of everything. Renaming a variable or a class has the same effect, Visual Studio will take care of everything.